packages/core/types/src/http/return/admin.ts

```diff
@@ -0,0 +1,54 @@
+export interface BaseReturnItem {
+  id: string
+  quantity: number
+  received_quantity: number
+  reason_id?: string
+  note?: string
+  item_id: string
+  return_id: string
+  metadata?: Record<string, unknown>
+}
+
+export interface AdminReturnResponse {
+  id: string
+  order_id: string
+  status?: string
+  exchange_id?: string
+  claim_id?: string
+  order_version: number
+  display_id: number
+  no_notification?: boolean
+  refund_amount?: number
+  items: BaseReturnItem[]
+}
+
+export interface AdminInitiateReturnRequest {
+  order_id: string
+  location_id?: string
+  description?: string
+  internal_note?: string
```

Comment:
discussion: we would need to send this info when we confirm a return request right? When we initiate a return request we don't know this info yet and later we can't update this fields.

Reply:
Yes, that's right – it will be send with the confirmation request. Also just discussed the reason with Seb. We will move these from the return level to the return item level, as we also talked about this morning. This is pending design.

Reply:
> we add a general update return request endpoint which would update note, location etc.?

Yeah, that's also possible. We will have the following endpoint:
```
POST /admin/returns/:id 

{ location_id?, no_notification?, metadata? }
```

Reply:
Good point.
And based on the message displayed when the location selected doesn't have stock associated with that location, maybe we need an extra endpoint to update the return location and return the result of this stock location check.